### PR TITLE
Mem converter

### PR DESCRIPTION
**Pull request by StefaniaDamian**
Added converter for memory, along with the tests. Also changed MemoryInfo struct to only store the amount in GB.

### CODE REVIEW

File: memconverter/converter.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+var mb float64 = 1000000
+var gb float64 = 1000000000
+var tb float64 = 1000000000000
+var pb float64 = 1000000000000000
+
+var kib float64 = 1024
+var mib float64 = 1048576
+var gib float64 = 1073741824
+var tib float64 = 1099511627776
+var pib float64 = 1125899906842624
```
Comment (ubschmidt2) [inline memconverter/converter.go:15]: These constants can be defined as constants. Example:

```
const (
    kb float64 = 1000
    mb float64 = kb * 1000
    gb float64 = mb * 1000
    tb float64 = gb * 1000
    pb float64 = pb * 1000
)

In the example above I've used multiplication with the previous value to make it easier to read. I think this makes sense to the base-2 units below, too.
```

File: memconverter/converter.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+func fromGB(num float64, unit string) (rez float64, success bool) {
```
Comment (ubschmidt2) [inline memconverter/converter.go:17]: I'm wondering if this method should rather be called `toGB`?

Comment (ubschmidt2) [inline memconverter/converter.go:17]: Please return an `error` instead of a `bool`.  

Useful:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

File: go.mod [hunk 3]
```diff
@@ -2,4 +2,4 @@ module github.com/googleinterns/terraform-cost-estimation
 
 go 1.15
 
-require github.com/hashicorp/terraform-json v0.5.0 // indirect
+require github.com/hashicorp/terraform-json v0.5.0 // indirect
```
Comment (ubschmidt2) [inline go.mod:5]: Nit: the change to this file looks unrelated.

Comment (StefaniaDamian) [reply go.mod:5]: Yeah, not sure why it appears as a change.
